api/lib/source/polkadotV0-source.js

```diff
@@ -363,7 +363,12 @@ class polkadotAPI {
     return filteredRewards
   }
 
-  async getRewards(delegatorAddress) {
+  async getRewards(delegatorAddress, fiatCurrency, withHeight) {
+    if (this.network.network_type !== 'polkadot' && withHeight) {
+      throw new Error(
+        'Rewards are only queryable per height in Polkadot networks'
```

Comment:
Well, technically they are Substrate networks, but `network_type` is called `polkadot`. So I don't know. Maybe we can just leave it like that